mypy/typeops.py

```diff
@@ -113,6 +113,8 @@ def tuple_fallback(typ: TupleType) -> Instance:
                 and unpacked_type.type.fullname == "builtins.tuple"
             ):
                 items.append(unpacked_type.args[0])
+            elif isinstance(unpacked_type, TupleType):
```

Comment:
IIRC the idea was that it should never be true when we call `tuple_fallback` - we have several places where things like `tuple[*tuple[X,Y]]` are normalized to a plain tuple, and the original intent was to get rid of such constructs as early as possible. This is fine as a quickfix, but I'm afraid that this will manifest again when such nested tuple is used elsewhere.